app.rb

```diff
@@ -1,9 +1,24 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    @peeps = Peep.all
+    erb :index
+  end
+
+  get '/new' do
+    erb :new
+  end
+
+  post '/' do
+    Peep.new(peep: params['message'])
+    redirect '/'
+  end
+
```

Comment:
Well structured file. It is clear what each get and post does and is easy to read!

Reply:
You're almost there with the routing but to be properly RESTful, all the Post-related routes should start `/posts`